client/extensions/woocommerce/woocommerce-services/views/shipping-label/label-purchase-modal/customs-step/item-row.js

```diff
@@ -0,0 +1,111 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import TextField from 'woocommerce/woocommerce-services/components/text-field';
+import {
+	setCustomsItemDescription,
+	setCustomsItemTariffNumber,
+	setCustomsItemOriginCountry,
+} from 'woocommerce/woocommerce-services/state/shipping-label/actions';
+import {
+	getShippingLabel,
+	isLoaded,
+	getFormErrors,
+} from 'woocommerce/woocommerce-services/state/shipping-label/selectors';
+import CountryDropdown from 'woocommerce/woocommerce-services/components/country-dropdown';
+
+const TariffCodeTitle = localize( ( { translate } ) =>
+	<span>{ translate( 'Tariff Code' ) } (<a href="https://hts.usitc.gov/"
+		target="_blank" rel="noopener noreferrer">{ translate( 'look up' ) }</a>)</span>
+);
+
+const ItemRow = ( props ) => {
+	const {
+		errors,
+		packageId,
+		productId,
+		translate,
+		description,
+		defaultDescription,
+		tariffNumber,
+		originCountry,
+		countriesData,
+	} = props;
+
+	return <div className="customs-step__item-row">
+		<TextField
+			id={ packageId + '_' + productId + '_description' }
+			className="customs-step__item-description-column"
+			title={ translate( 'Description' ) }
+			value={ description }
+			placeholder={ defaultDescription }
+			updateValue={ props.setCustomsItemDescription }
+			error={ errors.description } />
+		<CountryDropdown
+			id={ packageId + '_' + productId + '_originCountry' }
+			className="customs-step__item-country-column"
+			title={ translate( 'Origin country' ) }
```

Comment:
It's not obvious that "Origin country" means where the product is manufactured / assembled (vs where it's shipped from). Would be great to add an explanation.

Reply:
I've added a tooltip to that field in https://github.com/Automattic/wp-calypso/pull/24974/commits/d85e02ad6dac8f7e1272e0e7f0292f0992b24934
I can't think on a better tooltip-sized explanation. EasyPost just says the same:
> origin_country = Where the item was manufactured or assembled.

But of course, the laws governing where an item was "made" is several pages long :)